bin/run-integration-tests.sh

```diff
@@ -36,6 +36,7 @@ fi
 
 # Sauce Labs arguments
 if [ "${DRIVER}" = "SauceLabs" ]; then
+  exit 0
```

Comment:
Would this still allow us to run download and headless tests in the pipeline? Worth pushing for a test run to make sure?

Reply:
Thinking about it - might it be worth adding our most high traffic URLs to the headless tests just to assert a 200 response? This might be enough to catch at least some errors?

Reply:
I think en-US would probably be fine as nearly all pages use a common template.

We already have /firefox/new/ and /firefox/all/ covered by headless (I think), so maybe add:

- `/en-US/`
- `/firefox/`
- `/firefox/mobile/`
- `/firefox/browsers/`
- `/firefox/proucts/`
- `/firefox/accounts/`
- `/firefox/%version%/whatsnew/`
- `/firefox/installer-help/`
- `/firefox/releasenotes/`
- `/firefox/channel/desktop/`
- `/firefox/developer/`

Anything else product related?

Reply:
Looks like most of these actually are already covered by our headless tests. I think those may only be looking for the redirects though. Maybe we can make them test the final destination as well?

https://github.com/mozilla/bedrock/blob/f9f5df9248e88f64e8cf2f7783b40b98e542fd49/tests/redirects/test_urls.py#L27-L60

Reply:
Some of those urls are long gone and 404. If we can update the list to be more recent like above then great.

Reply:
You could also add to something like these tests if you didn’t want to change redirect tests https://github.com/mozilla/bedrock/blob/master/tests/functional/test_generated_pages.py

Reply:
I've added some tests and am running integration tests again.

Reply:
If we later wanted something comprehensive I could have it download the sitemap and crawl the whole thing. Seems a bit much for this but possibly useful if we have to scale back the other.